types/parse/index.d.ts

```diff
@@ -256,10 +256,15 @@ declare namespace Parse {
      */
     class File {
 
-        constructor(name: string, data: any, type?: string);
+        constructor(name: string, data: number[], type?: string);
```

Comment:
I recommend to use a union type instead of declaring the constructor multiple times.
```js
constructor(name: string, data: number[] | { base64: string } | ...
```

Reply:
Got it, tnx